mod/notifications/start.php

```diff
@@ -41,6 +41,10 @@ function notifications_plugin_init() {
 function notifications_page_handler($page) {
 
 	elgg_gatekeeper();
+
+	// Set the context to settings
+	elgg_set_context('settings');
+
```

Comment:
We could still, e.g., push common logic into a notifications_page_init() function and call it at the top of the resource views.